extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/multiplepersistenceunits/MultiplePersistenceUnitsPackageAnnotationsTest.java

```diff
@@ -75,7 +75,7 @@ public void testUser() {
 
         OtherUserInSubPackage savedOtherUserInSubPackage = usersEntityManager.find(OtherUserInSubPackage.class,
                 otherUserInSubPackage.getId());
-        assertEquals(savedOtherUserInSubPackage.getName(), savedOtherUserInSubPackage.getName());
+        assertEquals(otherUserInSubPackage.getName(), savedOtherUserInSubPackage.getName());
```

Comment:
Well, it's the best way to avoid failing tests!

Reply:
Yeah, I'd be surprised if this fails 🤣